client/analysis.html

```diff
@@ -17,8 +17,85 @@
                 <h1>Youtube Comment Analysis.</h1>
                 <ul>
                    <li>Enter a video name to see what people think!</li>
-                    <li>Click <a href="#">here</a> to learn how I analyze the comments.</li>
+                    <li>Click <a href="#info">here</a> to learn more.</li>
                 </ul>
+                <div onclick="window.location='#_'" class="lightbox" id="info">
+                    <div id="infoBox">
+                        <h1>Welcome to my project!</h1>
+                        <h3>Introduction</h3>
+                        <p>
+                           This is my portion of the 2020 Google STEP Capstone project.
+                           With this feature, you can enter in a Youtube video name or ID
+                           and get a solid estimate on how the video was received via
+                           comment analysis. 
+                        </p>
+
+                        <p>
+                           In order to achieve this, I took advantage of the 
+                            <a href="https://cloud.google.com/natural-language" target="_blank">Google Natural Language (NLP)</a> 
+                            and <a href="https://www.perspectiveapi.com/#/home" target="_blank">Google Perspective</a> APIs 
+                            to analyze comments returned to me from the
+                            <a href="https://developers.google.com/youtube/v3" target="_blank">Youtube Data API</a>.
+                        </p>
+                        <h3>Method</h3>
+                        <p>
+                            In order to ensure the most accurate analysis while handling the minimum amount
+                            of data, this tool takes 20 random comments from the Youtube API, aggregates them
+                            into a single string of sentences, and runs the chunk through both the NLP and
+                            Perspective APIs. Here is what each API is responsible for:
+                        </p>
+                        <!--Potentially, we will take the most popular comments.-->
+                        <p>
+                            NLP
+                            <ul>    
```

Comment:
Direct children of `<ul>` tags must be `<li>`s. If this is not actually a list, which seems to be the case, then just make it a paragraph in HTML and add a bullet point and indentation using CSS.